### PR TITLE
Updated winform sample to show raising of intents from .NET

### PR DESCRIPTION
**Pull request by andy-westacott**
Also updated the required version of the adapter

### CODE REVIEW

File: csharp/winform/OpenFin.Interop.Win.Sample/MainWindow.cs [hunk 1]
```diff
@@ -97,6 +98,7 @@ private void openFin_InteropConnected(object sender, EventArgs e)
                 connectToBrokerButton.Enabled = false;
                 createBrokerButton.Enabled = false;
                 interopBrokerInput.Enabled = false;
+                fireIntent.Enabled = true;
```
Comment (obany) [inline csharp/winform/OpenFin.Interop.Win.Sample/MainWindow.cs:101]: Could we make the fireIntent only enabled if context type is contact

File: csharp/winform/OpenFin.Interop.Win.Sample/OpenFinIntegration.cs [hunk 2]
```diff
@@ -146,5 +148,34 @@ private void Runtime_Disconnected(object sender, EventArgs e)
         {
             RuntimeDisconnected?.Invoke(this, EventArgs.Empty);
         }
+
+        public async void FireIntent()
```
Comment (obany) [inline csharp/winform/OpenFin.Interop.Win.Sample/OpenFinIntegration.cs:152]: Can we pass the selected contact from the Contacts drop down ?

**Review by obany — CHANGES_REQUESTED** [review]
Couple of small tweaks

**Review by johnman — APPROVED** [review]

File: csharp/winform/OpenFin.Interop.Win.Sample/MainWindow.cs (hunk 1 above)
Comment (andy-westacott) [reply csharp/winform/OpenFin.Interop.Win.Sample/MainWindow.cs]: done

File: csharp/winform/OpenFin.Interop.Win.Sample/OpenFinIntegration.cs (hunk 2 above)
Comment (andy-westacott) [reply csharp/winform/OpenFin.Interop.Win.Sample/OpenFinIntegration.cs]: done

[the remark this answers is not in the document]
Comment (andy-westacott) [reply csharp/winform/OpenFin.Interop.Win.Sample/OpenFinIntegration.cs]: yep.  sounds good.  Want to do this now or as a separate PR?

**Review by johnman — APPROVED** [review]

**Review by obany — APPROVED** [review]

**Pull request merged**